index.mako

```diff
@@ -330,8 +330,10 @@ repository's configuration.
         the hook passes.  _new in 1.6.0_.
 =r=
     =c= [`log_file`](_#config-log_file)
-    =c= (optional) if present, the hook output will additionally be written
-        to a file.
```

Comment:
instead of a comma splice how about `if present, the hook output will additionally be written to a file when the hook fails or [verbose](...) is true`